content/default.nix

```diff
@@ -3,13 +3,21 @@
 
 let
   pkgs = import nixpkgs {};
+  sources = import ../nix/sources.nix {};
+  nix-filter = import sources.nix-filter;
 
 in
 {
   html.all = pkgs.stdenv.mkDerivation {
     name = "content";
-    # TODO We only need content/, man/, scripts/
-    src = ../.;
+    src = nix-filter {
+      root = ../.;
+      include = [
+        "content"
+        "man"
+        "scripts"
```

Comment:
If I know exactly which script is needed, can I be more specific here, e.g. use `scripts/some-script.sh` ?

Reply:
Yup, we can also point out to specific files.

```nix
include = [
	"scripts/some-script.sh"
]
```